src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala

```diff
@@ -45,7 +48,7 @@ trait DateTuple {
           if (y.length != 4) {
             throw new Exception("Year must be 4 digits")
           }
-          new LocalDate(y.toInt, monthOption.getOrElse(throw new Exception("Month missing")).toInt, dayOption.getOrElse(throw new Exception("Day missing")).toInt)
+          new LocalDate(y.toInt, monthOption.getOrElse(throw new Exception("Month missing")).trim.toInt, dayOption.getOrElse(throw new Exception("Day missing")).trim.toInt)
```

Comment:
Yes, you're right - datetuple already does, but the test didn't test for this. I've amended